aea/crypto/helpers.py

```diff
@@ -143,3 +147,61 @@ def _create_ethereum_private_key() -> None:
     account = Account.create()
     with open(ETHEREUM_PRIVATE_KEY_FILE, "w+") as file:
         file.write(account.key.hex())
+
+
+def _try_generate_testnet_wealth(identifier: str, address: str) -> None:
+    """
+    Generate wealth on a testnet.
+
+    :param identifier: the identifier of the ledger
+    :param address: the address to check for
+    :return: None
+    """
+    try:
+        import requests
+        import json
+
+        if identifier == FETCHAI:
```

Comment:
should this be implemented/defined as abstract methods in the `Crypto/LedgerApis` classes? So we avoid if-else pattern and it is more scalable.

Reply:
No I don't think so. The issue is that these are highly specific implementations to the underlying test networks. There is no unified API for accessing faucets across different test networks on Ethereum.